app/views/webauthn_verifications/status.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class>
+  <!--[if lte IE 9]>
```

Comment:
I'm thinking we should drop this too. I'm curious what cut-off RubyGems has for browser support but IE9 is quite the vintage browser.